packages/tools/src/utilities/contours/updateContourPolyline.ts

```diff
@@ -0,0 +1,62 @@
+import { glMatrix } from 'gl-matrix';
+import { Types } from '@cornerstonejs/core';
+import type { ContourAnnotation } from '../../types';
+import type { ContourWindingDirection } from '../../types/ContourAnnotation';
+import * as math from '../math';
+
+/**
+ * Update the contour polyline data
+ * @param annotation - Contour annotation
+ * @param viewport - Viewport
+ * @param polylineData - Polyline data (points, winding direction and closed)
+ * @param transforms - Methods to convert points to/from canvas and world spaces
+ */
+export default function updateContourPolyline(
+  annotation: ContourAnnotation,
+  polylineData: {
+    points: Types.Point2[];
+    closed?: boolean;
+    windingDirection?: ContourWindingDirection;
+  },
+  transforms: {
+    canvasToWorld: (point: Types.Point2) => Types.Point3;
+  }
+) {
+  const { canvasToWorld } = transforms;
+  const { data } = annotation;
+  const { points: polyline } = polylineData;
+  let { closed, windingDirection } = polylineData;
+  const numPoints = polyline.length;
+  const polylineWorldPoints = new Array(numPoints);
+  const currentWindingDirection = math.polyline.getWindingDirection(polyline);
```

Comment:
If you already have the winding direction from the function parameters, shouldn't we utilize it?

Reply:
targetWinding